Jenkinsfile

```diff
@@ -187,6 +159,11 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
               utils.shwrap("""
               coreos-assembler buildupload s3 --acl=public-read ${s3_builddir}
               """)
+            } else if (!prod) {
+              // In devel mode without an S3 server, just archive in Jenkins
+              // itself. Otherwise there'd be no other way to retrieve the
+              // artifacts.
+              archiveArtifacts('builds/latest/*')
```

Comment:
so this will just show up in the jenkins web interface ? Would we need to make the size of the PVC much larger for jenkins in this case ?

Reply:
> so this will just show up in the jenkins web interface ?

Yup, exactly!

> Would we need to make the size of the PVC much larger for jenkins in this case ?

Yeah, at least for the Jenkins in the devel namespace? (Though also in the prod case in preparations for #31). I can adjust `HACKING.md` to reflect this.

I guess another approach is to store builds in the PVC only in the devel case (but still build in the workspace as done in this patch). E.g. something like `/srv/builds/$devel_prefix`?

Reply:
> I guess another approach is to store builds in the PVC only in the devel case (but still build in the workspace as done in this patch). E.g. something like `/srv/builds/$devel_prefix`?

I think I like that better, but i'm not sure. It's a lot easier for me to think about freeing up space in the PVC for coreos-assembler-claim2 than it is to conceptualize freeing up space in the jenkins PVC. Then again if we can maybe we should just get rid of the `coreos-assembler-claim2` for the dev case and only have one storage requirement.